test/publishing_api_v2_test.rb

```diff
@@ -1232,4 +1232,24 @@ def content_item_for_content_id(content_id, attrs = {})
       end
     end
   end
+
+  describe "content ID validation" do
+    [:get_content, :get_links, :get_linked_items, :discard_draft].each do |method|
```

Comment:
Do we favour `%i(...)` for literal symbol arrays as we do with literal string arrays?